RecoVertex/PrimaryVertexProducer/src/DAClusterizerInZT_vect.cc

```diff
@@ -1556,3 +1556,27 @@ void DAClusterizerInZT_vect::dump(const double beta, const vertex_t& y, const tr
   }
 #endif
 }
+
+void DAClusterizerInZT_vect::fillPSetDescription(edm::ParameterSetDescription& desc) {
+  desc.addUntracked<double>("zdumpcenter", 0.);
```

Comment:
You could simplify further by calling here the `DAClusterizerInZ_vect::fillPSetDescription()`, and only adding the 4D related parameters to such a "base" config

Reply:
Thanks @perrotta, addressed here: https://github.com/cms-sw/cmssw/pull/33104/commits/55536f059e694b9ca297978d9fbef8817bd7f684

Reply:
Thanks @perrotta, addressed here: https://github.com/cms-sw/cmssw/pull/33104/commits/55536f059e694b9ca297978d9fbef8817bd7f684